apis/opts/portbindings_test.go

```diff
@@ -40,15 +42,16 @@ func TestVerifyPortBinding(t *testing.T) {
 		portBindings types.PortMap
 	}
 	tests := []struct {
-		name    string
-		args    args
-		wantErr bool
+		name      string
+		args      args
+		wantErr   bool
+		expectErr error
 	}{
-		// TODO: Add test cases.
 		{name: "testCase1", args: args{portBindings: map[string][]types.PortBinding{"21/ftp": {types.PortBinding{HostIP: "", HostPort: "21"}}}}, wantErr: false},
```

Comment:
@chuanchang could we change the case name to be more specific? I don't think the `testCase1` is useful.

Reply:
testIpPort, testPortFTP, testPortTCP ?

Reply:
@fuweid the "testCase1" and "testCase2" originally exist in current code base, they are not added by me, in order to keep consistent with existent code, I added "testCase3" as new case.

```shell
-               // TODO: Add test cases.
                {name: "testCase1", args: args{ports: []string{"127.0.0.1:80:80"}}, want: map[string][]types.PortBinding{"80/tcp": {types.PortBinding{"127.0.0.1", "80"}}}, wantErr: false},
                {name: "testCase2", args: args{ports: []string{"26:22"}}, want: map[string][]types.PortBinding{"22/tcp": {types.PortBinding{"", "26"}}}, wantErr: false},
+               {name: "testCase3", args: args{ports: []string{"foo/bar"}}, expectErr: fmt.Errorf("Invalid containerPort: bar"), wantErr: true},
```
Of course, I can change case name together if it's necessary.